packer/generate-windows-vhd-publishing-info.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+
+required_env_vars=(
+    "CLASSIC_SA_CONNECTION_STRING"
+    "STORAGE_ACCT_BLOB_URL"
+    "VHD_NAME"
+)
+
+for v in "${required_env_vars[@]}"
+do
+    if [ -z "${!v}" ]; then
```

Comment:
I think you want, no?
```bash
if [ -z "${v}" ]; then
  # do stuff
fi
```

Reply:
"${!v}" expands the of ${v} when v is an enumerator.
I want to make sure the environment vars in the list have values.
It took me a lot of googling to find that and i'm pretty sure this is the behavior i wanted.

Reply:
https://stackoverflow.com/questions/14049057/bash-expand-variable-in-a-variable